analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/SymbolicExecutionRunnerTest.cs

```diff
@@ -327,9 +328,25 @@ public void Method()
 
         [TestMethod]
         public void Analyze_Severity_ExecutesWhenAll() =>
-            Verify(@"string s = null;   // Noncompliant    {{Message for SAll}}
-                                        // Noncompliant@-1 {{Message for SMain}}
-                    s.ToString();       // Noncompliant    {{'s' is null on at least one execution path.}} - rule S2259");
+            VerifyCode<TestSERunnerCS>(@"
```

Comment:
I moved to S2222. S2259 just caused too much trouble all the time.

Reply:
As [expected](https://github.com/SonarSource/sonar-dotnet/pull/5904#discussion_r929686955) 😄